build.gradle.kts

```diff
@@ -29,3 +33,9 @@ tasks.register<Exec>("installGitHooks") {
 
 val initialTaskNames: List<String> = project.gradle.startParameter.taskNames
 project.gradle.startParameter.setTaskNames(initialTaskNames + listOf("installGitHooks"))
+
+tasks.register("detektAll").configure {
+    dependsOn(gradle.includedBuild("build-logic").task(":detektAll"))
+    dependsOn(gradle.includedBuild("subprojects").task(":detektAll"))
+    dependsOn(gradle.includedBuild("samples").task(":detektAll"))
+}
```

Comment:
Can't we iterate through all included builds?
`getIncludedBuilds()`

Reply:
yep, fixed
thanks